### PR TITLE
 Add support for preset default taglist

### LINKED ISSUE #46: Add support for preset default taglist
**Is your feature request related to a problem? Please describe.**
I don't want to write every time the same tag.  

**Describe the solution you'd like**
I want to have the possibility do get a set of default tags

**Describe alternatives you've considered**
To write every time the same tag or copy-pasting

**Additional context**
POST /hosts/{hostname}/databases/{dbname}/tags/

### PR DESCRIPTION
**Pull request by cirelli94**
Fixed #46

### CODE REVIEW

File: api-service/controller/controller.go [hunk 1]
```diff
@@ -98,6 +98,8 @@ type APIControllerInterface interface {
 	GetExadataStorageErrorCountStatusStats(w http.ResponseWriter, r *http.Request)
 	// GetExadataPatchStatusStats return all statistics about the patch status of the exadata using the filters in the request
 	GetExadataPatchStatusStats(w http.ResponseWriter, r *http.Request)
+	//GetDefaultDatabasesTags return the default list of database tags from configuration
+	GetDefaultDatabasesTags(w http.ResponseWriter, r *http.Request)
```
Comment (amreo) [inline api-service/controller/controller.go]: Are you sure that the 's' of databases in GetDefaultDatabasesTags should be present?

File: api-service/controller/routing.go [hunk 2]
```diff
@@ -45,6 +45,7 @@ func SetupRoutesForAPIController(router *mux.Router, ctrl APIControllerInterface
 	router.HandleFunc("/environments", ctrl.ListEnvironments).Methods("GET")
 	router.HandleFunc("/clusters", ctrl.SearchClusters).Methods("GET")
 	router.HandleFunc("/databases", ctrl.SearchDatabases).Methods("GET")
+	router.HandleFunc("/databases/defaulttags", ctrl.GetDefaultDatabasesTags).Methods("GET")
```
Comment (amreo) [inline api-service/controller/routing.go]: Are you sure that "/databases/defaulttags" is the best name?
I'm thinking, in the future will be added a endpoint like /databases/{dbname} though aa db can only identified with hostname+dbname?
Can you add a dash between default and tags?
Can a endpoint like /setting/database-dafault-tag-choiches be a better idea?

**Review by amreo — CHANGES_REQUESTED** [review]

File: api-service/controller/controller.go (hunk 1 above)
Comment (cirelli94) [reply api-service/controller/controller.go]: I don't know. I thought about that: that tags are the default for all dbs, not only one!
If you prefer, it's the same to me :+1:

Comment (amreo) [reply api-service/controller/controller.go]: Go for GetDefaultDatabasesTags

File: api-service/controller/routing.go (hunk 2 above)
Comment (cirelli94) [reply api-service/controller/routing.go]: `/settings/database-dafault-tag-choiches` is surely a better idea!
I'm going to create a new controller and service for settings then!

Comment (amreo) [reply api-service/controller/routing.go]: Ok

**Review by amreo — APPROVED** [review]

**Pull request merged**